src/renderer/context/testSequencerWS.context.tsx

```diff
@@ -71,7 +73,11 @@ export function TestSequencerWSProvider({
 
   const mapToHandler: { [K in MsgState]: (data: BackendMsg) => void } = {
     TEST_SET_START: (data) => {
-      console.log("starting tests", data);
+      setBackendState("TEST_SET_START");
```

Comment:
this is just my preference, but if these guys were enums it would be so clean (up to you :) )

Reply:
Ahah, faaacts

Reply:
In TypeScript let's use a type union instead of an enum :)

Example:
```typescript
// Type union declaration
type MyType = number | string;
```

Reply:
Already Type Union. Syed and Joey have strong preference for them